espnet2/enh/layers/beamformer.py

```diff
@@ -34,39 +102,50 @@ def get_rtf(
     Note: 4) Normalization at the reference channel is not performed here.
 
     Args:
-        psd_speech (ComplexTensor): speech covariance matrix (..., F, C, C)
-        psd_noise (ComplexTensor): noise covariance matrix (..., F, C, C)
+        psd_speech (torch.complex64/ComplexTensor):
+            speech covariance matrix (..., F, C, C)
+        psd_noise (torch.complex64/ComplexTensor):
+            noise covariance matrix (..., F, C, C)
         reference_vector (torch.Tensor or int): (..., C) or scalar
         iterations (int): number of iterations in power method
         use_torch_solver (bool): Whether to use `solve` instead of `inverse`
     Returns:
-        rtf (ComplexTensor): (..., F, C, 1)
+        rtf (torch.complex64/ComplexTensor): (..., F, C, 1)
     """
+    if isinstance(psd_speech, ComplexTensor):
+        complex_wrapper = FC
```

Comment:
I think calling this `warpper` is a bit confusing as it is also used in `espnet2/enh/espnet_model.py` to note something different.

I would use something like `module` or `impl`.

Reply:
Thanks, I have renamed them to keep consistency.